services/src/pro/handlers/users.rs

```diff
@@ -772,6 +772,7 @@ mod tests {
     }
 
     #[tokio::test(flavor = "multi_thread", worker_threads = 1)]
+    #[serial]
```

Comment:
because a different test disables the user registration and then this test here can randomly fail.

Reply:
But if the context disables user registration, why is it disabled in another context?

Reply:
there is one test that manipulates the config, to test whether it takes effect in the register/login handler. And then there is this test that only works if the registration is enabled. That's why the two tests cannot run in parallel and must be #[serial]

Reply:
This was not my point. You now have to know which kinds of tests do something with register/login, since race conditions are somewhat guaranteed.

We should avoid these kinds of things if we have the chance. We cannot, sometimes, if some C library uses a global variable. But we should circumvent this in our code, I guess.

The questions is… why did this occur… now? :D

Reply:
> The question is… why did this occur… now? :D

idk